### DISCUSSION TITLE
article/2010-11-11-shixi/

### DISCUSSION THREAD

**Discussion by giscus** [gitlGl/gitlGl.github.io#9] [Show and tell]
# article/2010-11-11-shixi/

去年七八月份就在法院执行局实习过了，这次毕业实习继续去的是法院执行局。 也不知道当时怎么活过来的，现在看起来，似乎已是两重天。 这次毕业实习是一

http://localhost:1313/article/2010-11-11-shixi/

**Comment by gitlGl** [comment]
嘿嘿
